recipe_recommender/etl/index_bodybuilding.py

```diff
@@ -9,53 +9,58 @@
 from utils import DB_URI
 
 Base = declarative_base()
-#
-# class Recipes(Base):
-#     __tablename__ = 'recipes'
-#
-#     recipe_id = Column(String, primary_key=True)
-#     recipe_name = Column(String)
-#     description = Column(String)
-#     tag = Column(String)
+
+
+class Recipes(Base):
+    __tablename__ = 'recipes'
+
+    recipe_id = Column(String, primary_key=True)
+    recipe_name = Column(String)
+    description = Column(String)
+   #tag = Column(String)
 
 class Ingredients(Base):
     __tablename__ = 'ingredients'
 
     ingredient_id = Column(Integer, primary_key=True)
     ingredient = Column(String)
 
-# class RecipeIngredients(Base):
-#     __tablename__ = 'recipe_ingredients'
-#
-#     id = Column(BigInteger, primary_key=True)
-#
-#     recipe_id = Column(String, ForeignKey('recipes.recipe_id'))
-#     ingredient_id = Column(Integer, ForeignKey('ingredients.ingredient_id'))
-#     amount = Column(Float)
-#     unit = Column(String)
+
+class RecipeIngredients(Base):
+    __tablename__ = 'recipe_ingredients'
+
+    id = Column(BigInteger, primary_key=True)
+
+    # recipe_id = Column(String, ForeignKey('recipes.recipe_id'))
+    recipe_id = Column(String)
+    ingredient_id = Column(Integer, ForeignKey('ingredients.ingredient_id'))
+    amount = Column(Float)
+    unit = Column(String)
+    descriptor = Column(String)
+
 
 if __name__ == '__main__':
-    print('db uri ', DB_URI )
     engine = create_engine(DB_URI)
     session = Session(bind=engine)
-    # sessionmaker(bind=engine)
-
 
+    recipes = pd.read_csv('../../data/cleaned/recipes.csv')
```

Comment:
This should be changed to use the `os` module. Should be similar to `bodybuilding.py` (this is a PR now), where we use `os.path.dirname(os.path.abspath(__file__))`